### PR TITLE
Fix non-sortable questionnaires

### LINKED ISSUE #653: Contributor questionnaires not sortable
On the questionnaire page the contributor questionnaires are not sortable.

### PR DESCRIPTION
**Pull request by karyon**
Fixes #653

### CODE REVIEW

File: evap/static/js/sortable_query_binding.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/**
```
Comment (janno42) [inline evap/static/js/sortable_query_binding.js]: why do we need that?

Comment (karyon) [inline evap/static/js/sortable_query_binding.js]: to make those other changes possible. before i was getting raw DOM elements with get(), and now i can directly call sortable() on a jquery object. with this i also don't need to iterate over the two tables in the questionnaire index manually.

Comment (janno42) [inline evap/static/js/sortable_query_binding.js]: so much code. i especially like the `/* CODE */`. that's helpful. but you're the clean-up guy. so if you're okay with it, it's fine.

**Comment by janno42** [comment]
:+1:

File: evap/static/js/sortable_query_binding.js (hunk 1 above)
Comment (karyon) [inline evap/static/js/sortable_query_binding.js]: i'm not sure what you mean, this is a lib just like the core sortable.js or bootstrap...?

Comment (karyon) [inline evap/static/js/sortable_query_binding.js]: yeah i use only a few lines of this plugin but that's how using libs works.

**Pull request merged**